### ISSUE TITLE
Reports and other payment stats are off?

### ISSUE THREAD

**Issue by dannyvankooten** [easydigitaldownloads/Easy-Digital-Downloads#2815]
When manually querying the database I'm getting totally different results than the results EDD is giving me through the Reports graph or even through the Payments list table.

I import every single payment into my accounting software and am getting different results between the software and EDD. However, when manually querying the database it seems to give the same result as my accounting software does.

Am I missing something here? I've noticed this several times now.

```sql
# Query total earnings in June
SELECT sum(pm.meta_value) AS total 
FROM dvk_posts p, dvk_postmeta pm 
WHERE pm.post_id = p.id AND p.post_status = 'publish' AND p.post_type = 'edd_payment' 
AND pm.meta_key = '_edd_payment_total' 
AND p.post_date >= '2014-06-01 00:00:00' 
AND p.post_date <= '2014-06-30 23:59:59' 
```

**Comment by pippinsplugins** [comment]
I'm punting to 2.3.3 due to lack of feedback.

Note: this may be fixed by reducing the likelihood for race conditions in #2558.

We should test this further as well by using multiple instances of WP CLI to create hundreds of thousands of simultaneous purchase records.

**Comment by cklosowski** [comment]
Punting as we've had no feedback and #2558 was released which should reduce the chance of the race condition. We can keep an eye on it as things progress

**Comment by pippinsplugins** [comment]
I've seen further evidence that earnings stats for individual downloads is definitely wrong. It's possible it's left over from the possible race conditions improved in #2558.

**Comment by pippinsplugins** [comment]
Likely in part due to #3698

**Comment by pippinsplugins** [comment]
With the fixes made in #3698, I believe this is finally resolved.

I'm going to close this now. If there are any further reports of this after recounting stats (see #3633) we can re-open.
